### PR TITLE
Docs: Adds section on Querying Logs for InfluxDB

### LINKED ISSUE #17715: InfluxDB: Document how to visualize logs in explore
Updates needed of [Using InfluxDB in Grafana](https://grafana.com/docs/features/datasources/influxdb/) due to #17298:
- Add a new "Querying logs" section similar to how what we have for Loki, see [docs](https://grafana.com/docs/features/datasources/loki/#querying-logs).

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #17715

**Special notes for your reviewer**:
First time I added documentation and images so please advise if I'm doing something in the wrong order.

### CODE REVIEW

**Comment by hugohaggmark** [comment]
Just saw that vscode auto formatted stuff, I'll revert the commit and add new section again.

File: docs/sources/features/datasources/influxdb.md [hunk 1]
```diff
@@ -117,6 +117,26 @@ You can switch to raw query mode by clicking hamburger icon and then `Switch edi
 You can remove the group by time by clicking on the `time` part and then the `x` icon. You can
 change the option `Format As` to `Table` if you want to show raw data in the `Table` panel.
 
+## Querying Logs (BETA)
+
+Querying and displaying log data from InfluxDB is available via [Explore](/features/explore).
+Select the InfluxDB data source, change to Logs using the Metrics/Logs switcher, <br>
+![](/img/docs/influxdb/explore_metrics_logs_switch.png<br>
```
Comment (kaydelaney) [inline docs/sources/features/datasources/influxdb.md]: Missing a closing parentheses:
```suggestion
![](/img/docs/influxdb/explore_metrics_logs_switch.png)<br>
```

File: docs/sources/features/datasources/influxdb.md [hunk 2]
```diff
@@ -117,6 +117,26 @@ You can switch to raw query mode by clicking hamburger icon and then `Switch edi
...
+and then use the `Measurements/Fields` button <br>
+![](/img/docs/influxdb/explore_measurements_fields_button.png<br>
```
Comment (kaydelaney) [inline docs/sources/features/datasources/influxdb.md]: See above
```suggestion
![](/img/docs/influxdb/explore_measurements_fields_button.png)<br>
```

File: docs/sources/features/datasources/influxdb.md [hunk 3]
```diff
@@ -117,6 +117,26 @@ You can switch to raw query mode by clicking hamburger icon and then `Switch edi
...
+to display your logs.
+
+### Log Queries
+
+The Logs Explorer (the `Measurements/Fields` button) next to the query field shows a list of measurements and fields. Choose the desired measurement that contains your log data and then choose which field Explore should use to display the log message.
+<br>
+![](/img/docs/influxdb/explore_measurements_fields_button.png<br>
```
Comment (kaydelaney) [inline docs/sources/features/datasources/influxdb.md]: See above
```suggestion
![](/img/docs/influxdb/explore_measurements_fields_button.png)<br>
```

File: docs/sources/features/datasources/influxdb.md (hunk 1 above)
Comment (hugohaggmark) [reply docs/sources/features/datasources/influxdb.md]: Thanks @kaydelaney

**Review by marefr — CHANGES_REQUESTED** [review]
Think that it would be nicer to have one big screenshot that shows query field, date histogram and some log messages.

Then you can probably remove `<br>` which currently breaks apart the text.

**Comment by hugohaggmark** [comment]
> Think that it would be nicer to have one big screenshot that shows query field, date histogram and some log messages.
> 
> Then you can probably remove `<br>` which currently breaks apart the text.

Thanks, could you be a bit more specific. Should I remove all the pictures replacing it with the one you suggest or are you thinking of a specific paragraph?

**Comment by marefr** [comment]
Sync with @kaydelaney she has done similar changes for influx. Basically just one screenshot.

**Review by marefr — CHANGES_REQUESTED** [review]
Approved, but blocked by https://github.com/grafana/website/pull/217

**Review by marefr — APPROVED** [review]

**Pull request merged**